src/main/kotlin/no/nav/syfo/dialogmote/domain/DialogmoteInnkallingTekstEnum.kt

```diff
@@ -0,0 +1,30 @@
+package no.nav.syfo.dialogmote.domain
+
+enum class DialogmoteInnkallingTekstEnum {
+    INNKALLING_SM_OVERSKRIFT,
```

Comment:
Pirk: Erstatte sykmeldt(SM) med arbeidstaker(AT).
Vi gikk litt bort ifra å omtale en sykmeldt arbeidstaker som sykmeldt og heller bruke ordet arbeidstaker i løsningen, da er det fint om koden også gjør(pluss for konsistens).